examples/measure_methods/fromFile_ADCIRCMap/BET_Q_1D.py

```diff
@@ -8,7 +8,7 @@
 import bet.sample as sample
 
 # Import "Truth"
-mdat = sio.loadmat('../matfiles/Q_2D')
+mdat = sio.loadmat('../Q_2D')
```

Comment:
```suggestion
mdat = sio.loadmat('Q_2D')
```

The data file was moved from a separate directory into the same path as the example. Changing to the above fixes a load error.